### ISSUE TITLE
ASNPOOL keyword should include the extension

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#6680]
_Issue [JP-2494](https://jira.stsci.edu/browse/JP-2494) was created on JIRA by [Lisa Gardner](https://jira.stsci.edu/secure/ViewProfile.jspa?name=gardner):_

The ASNPOOL keyword should include the file extension.
Example: jw00625_20210713t194701_pool is the value ASNPOOL in the FITS header.

The file name of the actual Association Pool file being ingested would be like jw00625_20210713t194701_pool.csv. 

The ASNTABLE keyword does include the the extension so the change to ASNPOOL keyword will make it consistent with the ASNTABLE keyword.

**Comment by stscijgbot-jp** [comment]
_Comment by [Linda Thompson](https://jira.stsci.edu/secure/ViewProfile.jspa?name=lthomp) on [JIRA](https://jira.stsci.edu/browse/JP-2494?focusedCommentId=566839#comment-566839):_

I cannot change the ASNPOOL value. It is not set in SDP code (nor is ASNTABLE).

from core.association.schema.json:
```
    "pool_name" : {
        "fits_keyword" : "ASNPOOL",
        "example" : "jw12345_20140626T103556_pool.csv",
        "source" : "Calibration Software",
        "level" : "2b",
    "table_name" : {
        "fits_keyword" : "ASNTABLE",
        "example" : "jw84500-001_20150820T170446_spectrum_001_asn.json",
        "source" : "Calibration Software",
        "level" : "2b",
```

I moved the ticket to JP and removed myself as the Assignee.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-2494?focusedCommentId=568886#comment-568886):_

Fixed  by #6699
